cli/cmd/lib_cluster_config.go

```diff
@@ -79,40 +79,40 @@ func readUserClusterConfigFile(clusterConfig *clusterconfig.ClusterConfig) error
 }
 
 func getAccessClusterConfig() (*clusterconfig.AccessClusterConfig, error) {
```

Comment:
nit: should this be renamed to getAccessClusterConfig -> getAccessConfig and similarly clusterconfig. AccessClusterConfig -> clusterconfig.AccessConfig?